### DISCUSSION TITLE
DetailsList with row actions

### DISCUSSION THREAD

**Discussion by Rocky-Moon** [microsoft/fluentui#15980] [General] [answered]
A common UI case when using a DetailsList is to be able to perform actions on the row in the details list.

There are no current examples within the DetailsList on options doing this, is there already built in functionality or is it a roll your own scenario?  Would be good to have a standardised approach to this.

**Comment by JustSlone** [comment, marked as answer]
There is the onItemInvoked prop that takes a callback that gets called when the user double clicks or presses enter on a focused row. 

You can see an example here:
https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/basic

Is that what you are looking for?
